### ISSUE TITLE
Unable to get any info from /proc/cpuinfo on Intel x5650 based server system

### ISSUE THREAD

**Issue by DevBennyturtle** [oshi/oshi#2230]
This might sound like the wrong place, hear me out first. 
I run a minecraft server and have decent c++ and python skills, and the server refused to start.
I went into the debug log because the console output showed nothing, and saw that oshi had an issue. 
I tried googling the issue, and I believe I am the only one to have said issue so far, as most consumers "hosting" minecraft servers either use a VPS provider or off the shelf hardware, not weird real server configurations.
I have a Dell Poweredge R710 with 2 Intel X5650 CPU's, and 64GB (/ 72GB max) of ram (2 dimms died, don't judge)
I wanted to utilize some of the spare cpu time and spun up a kvm with virt-install, and downloaded minecraft server to host.
![image](https://user-images.githubusercontent.com/45775493/198888381-8240a46f-ee2a-4f81-a879-e8cb0edec966.png)
I get that when trying to run the modded server. I think: ok, someone must have had a similar issue somewhere. Nope.
There are sparse reports of issues with having 2 .properties files, and there usually is a fix, but oshi can't even understand what system I'm running on at the moment. its a full KVM of my R710 server, with little to no modifications in anything cpu related.
I am completely unsure how to proceed, and at this point am fairly sure its a core issue here as for some reason oshi can't even comprehend what system I'm running on.

Thank you in advance for any help if any at all you can provide, I'm just confused and desperate as to why this just collapsed on me.

Unsure if this will help, but here is my proc/cpuinfo file (for at least one core)
![image](https://user-images.githubusercontent.com/45775493/198888572-d26fbec6-f560-4077-a1e8-c277eae7754e.png)

**Comment by dbwiddis** [comment]
The log shows it’s not finding at least one properties file on the class path. It likely is missing the config file which includes the “/proc” prefix.   You can work around that using GlobalConfig.  

But a better fix is to find out why it is not reading the config files.  Are you using the latest version?

**Comment by DevBennyturtle** [comment]
I am fairly confident I fixed it. It wasn't the main issue but by fixing it I was able to get just to the point of seeing the core issue. For anyone using this for minecraft servers, what I did was fake /proc/cpuinfo as mine wasnt working right, and forge went past it which let me see entitiyculling was messing up.
If anyone has a similar issue, just run (mount --bind /whatever/path/cpuinfo/from/another/linux/box /proc/cpuinfo) and it will get past it just enough to see the main issue.
dbwiddis, thank you for trying to help, but anyone who has this issue wouldn't be able to find that helpful, as this is from minecraft. I did not write or even see a bit of the code for the minecraft server mods I'm running, I am simply running and trying to diagnose them.
